tests/test_synced_collections/synced_collection_test.py

```diff
@@ -398,6 +398,23 @@ class A:
             with pytest.raises(TypeError):
                 synced_collection[key] = testdata
 
+    def test_multithreaded(self, synced_collection):
```

Comment:
Also needs tests that use the `enable_multithreading` / `disable_multithreading` methods.

Reply:
Done.